src/main/java/event/EventList.java

```diff
@@ -0,0 +1,162 @@
+package event;
+
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import ui.Ui;
+
+/**
+ * Handles all functions related to the list of events.
+ */
+public class EventList {
+
+    Ui ui = new Ui();
+
+    /** Stores the event information. */
+    private ArrayList<Event> events;
+
+    /**
+     * Constructor for the EventList class.
+     */
+    public EventList() {
+        events = new ArrayList<Event>();
+    }
+
+    /**
+     * Adds a new event to the list.
+     *
+     * @param newEvent Event represents the new event tot be added.
+     */
+    public void addEvent(Event newEvent) {
+        ui.printLine();
+        System.out.println("A new event with the following information has been added.");
```

Comment:
Please put this message after the event has been successful added

Reply:
Ok, will update as required.

Reply:
is ok ive done it